pages/agent/v3/prioritization.md

```diff
@@ -25,6 +25,9 @@ or with the `BUILDKITE_AGENT_PRIORITY` an environment variable:
 env BUILDKITE_AGENT_PRIORITY=9 buildkite-agent start
 ```
 
+> 📘
+> Using the `--priority` flag requires agent 3.65.0 or higher.
```

Comment:
Nitpicks: We normally include "v" for version before the number, and use "later":

```suggestion
> Using the `--priority` flag requires agent v3.65.0 or later.
```